docs/standard/glossary.md

```diff
@@ -0,0 +1,267 @@
+
+See [Intro to .NET Native and CoreRT](https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md)
+
+## ecosystem
+
+All of the runtime software, development tools, and community resources that are used to build and run applications for a given technology.
+
+The term ".NET ecosystem" differs from similar terms such as ".NET stack" in its inclusion of third-party apps and libraries. Here's an example in a sentence:
+
+- "The motivation behind the [.NET Standard](#net-standard) is to establish greater uniformity in the .NET ecosystem." 
+
+## framework
+
+In general, a comprehensive library that facilitates development and deployment of applications that are based on a particular technology. In this general sense, ASP.NET Core and Windows Forms are examples of application frameworks.
+
+The word "framework" has a more specific technical meaning in the following terms:
+* [.NET Framework](#net-framework)
+* [target framework](#target-framework)
+* [TFM (target framework moniker)](#tfm)
+
+In the existing documentation, "framework" sometimes refers to an [implementation of .NET](#implementation-of-net). For example, an article may call .NET Core a framework. We plan to eliminate this confusing usage from the documentation.
+
+## GC
+
+Garbage collector.
+
+The garbage collector is an implementation of automatic memory management.  The GC frees memory occupied by objects that are no longer in use. 
+
+See [Garbage Collection](garbage-collection.md).
+
+## IL
+
+Intermediate language.
+
+Higher-level .NET languages, such as C#, compile down to a hardware-agnostic instruction set, which is called Intermediate Language (IL). IL is sometimes referred to as MSIL (Microsoft IL) or CIL (Common IL).
+
+## JIT
+
+Just-in-time compiler.
+
+This technology compiles [IL](#il) to machine code that the processor understands. It's called JIT because compilation happens on demand and is performed on the same machine that the code needs to run on. Since JIT compilation occurs during execution of the application, compile time is part of the run time. Thus, JIT compilers have to balance time spent optimizing code against the savings that the resulting code can produce. But a JIT knows the actual hardware and can free developers from having to ship different implementations.
+
+## implementation of .NET
+
+An implementation of .NET includes the following:
+
+- One or more runtimes. Examples: CLR, CoreCLR, CoreRT.
```

Comment:
@terrajobst suggested "one or more".  Maybe an instance of an implementation would only have one? 😄